hr_employee_document/models/hr_employee_public.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeePublic(models.Model):
+    _inherit = "hr.employee.public"
+
+    document_ids = fields.One2many(
+        comodel_name="ir.attachment",
+        compute="_compute_document_ids",
+        string="Documents",
+    )
+    documents_count = fields.Integer(
+        compute="_compute_document_ids", string="Document Count",
+    )
+    is_logged = fields.Boolean(compute="_compute_is_logged", store=False)
+
+    def _compute_document_ids(self):
+        attachments = self.env["ir.attachment"].search(
+            [("res_model", "=", "hr.employee"), ("res_id", "in", self.ids)]
+        )
+
+        result = dict.fromkeys(self.ids, self.env["ir.attachment"])
+        for attachment in attachments:
+            result[attachment.res_id] |= attachment
+
+        for employee in self:
+            employee.document_ids = result[employee.id]
+            employee.documents_count = len(employee.document_ids)
+
+    def _compute_is_logged(self):
+        for item in self:
+            item.is_logged = False
```

Comment:
You can do outside the loop (new ORM allows it):
```suggestion
        self.is_logged = False
```